app/client/src/assets/sass/pages/_app.scss

```diff
@@ -1,8 +1,7 @@
 .app {
-  height: 90vh;
+  height: 100vh;
+  overflow-y: auto;
   &__view {
-    overflow: auto;
-    height: 90vh;
     padding: 2rem;
 
     @media screen and (max-width: 375px) {
```

Comment:
Nice media query but I'm wondering if we should increase to 500px?

Reply:
This is going to change eventually, want to have the `.app` div to be the entire viewport width and height and have the `.app__view` div be the scrollable dashboard content for the registration form and other components to be built.